packages/flutter/lib/src/foundation/change_notifier.dart

```diff
@@ -558,3 +558,71 @@ class ValueNotifier<T> extends ChangeNotifier implements ValueListenable<T> {
   @override
   String toString() => '${describeIdentity(this)}($value)';
 }
+
+/// A [ValueListenable] that computes its value from other listeners.
+///
+/// This notifier listens to a list of other listeners and calls a compute function
+/// whenever any of them change to determine its own value.
+///
+/// The listeners are merged into a single listenable using [Listenable.merge].
+///
+/// ## Example
+///
+/// ```dart
+/// final ValueNotifier<int> a = ValueNotifier<int>(1);
+/// final ValueNotifier<int> b = ValueNotifier<int>(2);
+///
+/// final ComputedNotifier<int> sum = ComputedNotifier<int>(
+///   [a, b, c, d],
```

Comment:
Why listen to c and d if only a and b are used in compute?

Reply:
fixed